internal/api/api.go

```diff
@@ -59,3 +62,14 @@ func configureV1Router(router *mux.Router, runnerManager runner.Manager, environ
 		configureRoutes(v1)
 	}
 }
+
+// Version handles the version route.
+// It responds the release information stored in the configuration.
```

Comment:
Do we need to add the version as an additional key-value-pair to the example configuration?

Reply:
It is done automatically by our deployment playbook [cf704180b2c96118ad835a2805eb02f0029c91d0](https://gitlab.hpi.de/maximilian.pass/poseidon-deployment/-/commit/cf704180b2c96118ad835a2805eb02f0029c91d0)
But I now added a comment for that in the description. Thank you for pointing out this ambiguity.

https://github.com/openHPI/poseidon/blob/10754d778d83cd11403053e58797678c89c3ff0a/configuration.example.yaml#L56-L58

Reply:
That's awesome, thanks! 🙏